float.css

```diff
@@ -1,6 +1,7 @@
 *{
```

Comment:
That's true. But what's the alternative?

On Monday, August 22, 2016, khanh nguyen <notifications@github.com> wrote:

> In float.css
> <https://github.com/Jenn083/Jane-Doe/pull/1#discussion_r75738629>:
>
> > @@ -1,6 +1,7 @@
> >  *{
>
> universal selector is a feature that is created. It is not bad when you
> are applying to it's feature. Take a look at the post from google engineer
> explain the benefit of using universal selector :
> http://www.paulirish.com/2012/box-sizing-border-box-ftw/ . Now why we
> shouldn't use it in general. Mostly, there might be a duplicated styling or
> if you don't intend to have default styling element as you put in the
> universal selector. It will be harder for you to find out why the issue is
> happening. The performance impact is depend on the browser. (mostly IE
> 6,7). I think you can try to use this technique but I would recommend to
> use it carefully. Know when you need to use it. The feature is created for
> reason.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Jenn083/Jane-Doe/pull/1/files/84e5654ea8afd470360afaa0c9c4464ee1ea1179#r75738629>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHHcIMM0fmwR9M4mkaJByZSaRwzWdeYSks5qifMLgaJpZM4Jp2_6>
> .
>